src/ServiceStack.Client/UrlExtensions.cs

```diff
@@ -481,7 +481,7 @@ internal static string GetQueryString(object request, IDictionary<string, RouteM
                     continue;
 
                 var value = queryProperty.Value.GetValue(request, true);
-                if (value == null)
+                if (value == null || (value is Array && value.ToJsv() == EmptyArray))
```

Comment:
Note serializing an object to detect its serialized state is inefficient, can use `value is Array array && array.Length == 0` instead.